### PR TITLE
feat: prefill campaign application organizer form values from current user (Epic #1842)

### PR DESCRIPTION
**Pull request by teodorazhelyazkova**
Prefill campaign application organizer form values from current user as requested in the following task https://github.com/users/gparlakov/projects/1/views/1?pane=issue&itemId=68583764

*Connected to [Epic #1842](https://github.com/podkrepi-bg/frontend/issues/1842)

### Steps to test

1. Login as normal user
2. Visit `http://localhost:3040/campaigns/application`

### CODE REVIEW

File: src/components/client/campaign-application/CampaignApplicationForm.tsx [hunk 1]
```diff
@@ -51,9 +40,27 @@ const steps: StepType[] = [
...
 
-export default function CampaignApplicationForm() {
+type Props = {
+  person: PersonResponse
+}
+
+export default function CampaignApplicationForm({ person }: Props) {
   const [activeStep, setActiveStep] = useState<Steps>(Steps.ORGANIZER)
 
+  const initialValues: CampaignApplicationFormData = useMemo(
```
Comment (sashko9807) [inline src/components/client/campaign-application/CampaignApplicationForm.tsx:50]: useMemo isn't really needed for this case
 We don't do any calculations here.

File: src/components/client/campaign-application/CampaignApplicationPage.tsx [hunk 2]
```diff
@@ -1,10 +1,24 @@
+import { CircularProgress, Grid } from '@mui/material'
+import { useSession } from 'next-auth/react'
+import { useViewPersonByKeylockId } from 'common/hooks/person'
 import Layout from '../layout/Layout'
 import CampaignApplicationForm from './CampaignApplicationForm'
 
 export default function CampaignApplicationPage() {
+  const { data: session } = useSession()
+  const { data: person, isLoading } = useViewPersonByKeylockId(session?.user?.sub as string)
```
Comment (sashko9807) [inline src/components/client/campaign-application/CampaignApplicationPage.tsx:9]: Check useCurrentPerson hook, should do the exact same job.

**Review by sashko9807 — CHANGES_REQUESTED** [review]

File: src/components/client/campaign-application/CampaignApplicationPage.tsx (hunk 2 above)
Comment (teodorazhelyazkova) [reply src/components/client/campaign-application/CampaignApplicationPage.tsx]: Yep, done

File: src/components/client/campaign-application/CampaignApplicationForm.tsx (hunk 1 above)
Comment (teodorazhelyazkova) [reply src/components/client/campaign-application/CampaignApplicationForm.tsx]: Okay, removed

**Review by sashko9807 — APPROVED** [review]

**Pull request merged**